miio/huizuo.py

```diff
@@ -34,21 +120,6 @@ class HuizuoException(DeviceException):
 
 class HuizuoStatus:
     def __init__(self, data: Dict[str, Any]) -> None:
-        """
```

Comment:
Could you please re-add an example response (with most possible properties, if possible)?

Reply:
Also, add a note that if the device does not have the property, `None` will be returned (all those optionals) :-)

Reply:
Returned the example response, but only for one type of lamp (which I have).
Also, additional note was added